### PR TITLE
Front-end Student Profile 

### PR DESCRIPTION
**Pull request by yahyaHB**
Did the student profile with the front-end
and still has some modification to do as well,
related to issue #3
:heart:

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
-      </Switch>
-    </div>
-  </BrowserRouter>,
+  <Provider store={store}>
+    <BrowserRouter>
+      <div className='app'>
+        <Switch>
+          <Route exact path='/' component={Home} />
+          <Route exact path='/profile' component={StudentProfile} />
+          <Redirect to='/' />
```
Comment (MahmoudMH) [inline src/client/index.js]: remove this line, this was for HashRouter

File: src/client/components/search/Search.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+import './Search.css';
+
+class Search extends Component {
+  constructor() {
+    super();
+    this.onSubmit = this.onSubmit.bind(this);
+  };
+  onSubmit(e) {
```
Comment (MahmoudMH) [inline src/client/components/search/Search.js]: can you understand what this function does, please?

File: src/client/components/footer/FooterStyle.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#footer {
+clear: both;
```
Comment (astroash) [inline src/client/components/footer/FooterStyle.css]: the spacing of your css is misleading, this should be indented

File: src/client/components/footer/FooterStyle.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+width: 100%;
+height: 105px;
+background-color: #fff;
+}
+
+.footer-center {
+	width: 100%;
```
Comment (astroash) [inline src/client/components/footer/FooterStyle.css]: This has too much indentation.

File: src/client/components/studentProfile/StudentProfile/StudentProfile.css [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+  top: 383px;
+}
+
+.gitterbtn {
+  color: #ED2067;
+  padding: 8px;
+}
+
+.studentPurpose {
+position: absolute;
```
Comment (astroash) [inline src/client/components/studentProfile/StudentProfile/StudentProfile.css]: missing indentation

File: src/client/index.js (hunk 1 above)
Comment (astroash) [reply src/client/index.js]: Has this comment been addressed?

Comment (astroash) [reply src/client/index.js]: Has this comment been addressed?

**Comment by astroash** [comment]
@yahyaHB can you confirm is @MahmoudMH's question above has been answered.

**Comment by yahyaHB** [comment]
@astroash yes, it is answered

**Comment by astroash** [comment]
@yahyaHB can you please type responses to questions out on github so that it is clear that they have been answered

File: src/client/index.js (hunk 1 above)
Comment (yahyaHB) [reply src/client/index.js]: it's been used in this branch but modified in the new one 
so once it's merged the new PR will override it

File: src/client/components/search/Search.js (hunk 2 above)
Comment (yahyaHB) [reply src/client/components/search/Search.js]: when the submit button has been clicked, it will run function to send some query to fetch some data

**Comment by yahyaHB** [comment]
@astroash thanks :heart:

**Pull request merged**